apps/ui/src/components/Ui/Button.vue

```diff
@@ -27,7 +27,7 @@ withDefaults(
   </button>
 </template>
 
-<style scoped lang="scss">
```

Comment:
Why is it no longer scoped? That's generally not good because it will leak styles.

Reply:
For some reason, the UILoading icon can not be styled when inside the button with the `scope`

Reply:
Can you link to where do you use it like this (couldn't find it in the diff) so I can take a look?

Reply:
It's to fix the loading icon, on the vote modal CONFIRM button, when pressed, and while waiting for wallet confirmation